tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
   "compilerOptions": {
     "target": "es2019",
+    // Use ES2024.collection, otherwise mobx 6.13 fails with:
```

Comment:
The mobx tickets suggest to set `skipLibCheck` to true, but I'm not convinced that less static checking is what we want. The tickets for this issue are from mid-2024, so as second option they suggest setting `lib` to `ESNext` since that was the only place for  `ReadonlySetLike` at the time.

Since `ReadonlySetLike` is also present in `ES2024.collection` now, I opted for that, and kept `ES2019` for the rest. For some reason StoryBody.tsx failed to compile then because of a missing iterator in some HTMLCollection, so added `DOM.Iterable` to fix that.